path.go

```diff
@@ -0,0 +1,47 @@
+package httpapi
+
+import (
+	"context"
+
+	cid "github.com/ipfs/go-cid"
+	ipld "github.com/ipfs/go-ipld-format"
+	ipfspath "github.com/ipfs/go-path"
+	"github.com/ipfs/interface-go-ipfs-core"
+)
+
+func (api *HttpApi) ResolvePath(ctx context.Context, path iface.Path) (iface.ResolvedPath, error) {
+	var out struct {
+		Cid     cid.Cid
+		RemPath string
+	}
+
+	//TODO: this is hacky, fixing https://github.com/ipfs/go-ipfs/issues/5703 would help
```

Comment:
Can we file a bug for this in this repo?

Reply:
https://github.com/ipfs/go-ipfs-http-client/issues/2